pom.xml

```diff
@@ -71,8 +77,18 @@
             <version>1.5.1</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>junit</artifactId>
+            <version>1.19</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
-
+    <properties>
+        <jenkins.version>2.32</jenkins.version>
```

Comment:
I just moved this stuff inside because `mvn:hpi run` had problems otherwise. But I could start `mvn hpi:run` with parameters instead of defining it in the pom.